src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -350,8 +414,28 @@ private void InsertEntry(NodeSearchElementViewModel entry, IEnumerable<string> c
             NodeSearchElementViewModel entry, NodeCategoryViewModel target,
```

Comment:
`PlaceInNewCategory` should be renamed as `InsertEntryIntoCategory` or something like that. And then the first parameter should be `NodeCategoryViewModel category`, and then the second parameter should be `NodeSearchElementViewModel entry` (to signify the relationship between a category and an entry).